package.json

```diff
@@ -11,15 +11,16 @@
     "dist"
   ],
   "scripts": {
-    "prepare": "modern build",
+    "prepare": "publint && modern build",
```

Comment:
publint will check main assets if exist , can we put it after build like this: modern build && publint

Reply:
sure!